src/components/Course/Course.vue

```diff
@@ -1,96 +1,109 @@
 <template>
-  <div
-    :class="{
-      'course--min': compact,
-      conflict: isCourseConflict(courseObj.uniqueID),
-      active: active,
-    }"
-    class="course"
-  >
-    <save-course-modal
-      :courseCode="courseCode"
-      @close-save-course-modal="closeSaveCourseModal"
-      @save-course="saveCourse"
-      @add-collection="addCollection"
-      v-if="isSaveCourseOpen"
-    />
-    <edit-color
-      :editedColor="editedColor"
-      @color-course="colorCourse"
-      @color-subject="colorSubject"
-      @close-edit-color="closeEditColorModal"
-      v-if="isEditColorOpen"
-    />
+  <div class="course-container">
```

Comment:
I know it was my suggestion to rearrange, but I would just be careful with the class names here post-reshuffling. It looks like the CSS logic for some of the z-indices gets in a kerfuffle on account of this change, causing the modal to be hidden under the next course card.

<img width="230" alt="image" src="https://github.com/user-attachments/assets/676218f8-cd49-4a21-810e-bb5f4f7961c1">

An easy fix would be to reduce the `z-index` from 1 to 0 for the course card. Can be a part of the scope of frontend fixes branched off this PR.